.github/workflows/test_pull_requests.yml

```diff
@@ -21,6 +21,8 @@ jobs:
       - name: Check Manifest
         run: |
           pip install --upgrade pip
+          pip install -e .
+          python -c 'import napari.view_layers'
```

Comment:
this line will cause the failure of this check if not change the check manifest configuration, but it does not look like long-term solution. Maybe `make typestubs` should be called?
@goanpeca

Reply:
Yes, maybe that is better to use "make typestups"